wire-ios-share-engine/WireShareEngineTests/SharingSessionTests.swift

```diff
@@ -64,4 +65,22 @@ class SharingSessionTests: BaseSharingSessionTests {
         let conversations = sharingSession.writebleArchivedConversations.map { $0 as! ZMConversation }
         XCTAssertEqual(conversations, [archivedConversation])
     }
+
+    // MARK: - Init
+
+    func test_ItDoesNotInit_WhenCryptoboxMigrationIsPending() throws {
+        do {
+            // Given
+            mockCryptoboxMigrationManager.isMigrationNeeded = true
+
+            // When
+            _ = try createSharingSession()
+        } catch SharingSession.InitializationError.pendingCryptoboxMigration {
+            // Then
+            return
+        } catch {
+            XCTFail("unexpected error: \(error.localizedDescription)")
```

Comment:
this only fails if another error is thrown, should we also fail if no error is being thrown?

Reply:
done